startup_scripts/startup_script_utils/utils.py

```diff
@@ -0,0 +1,15 @@
+from typing import Tuple
+
+
+def split_params(params: dict, unique_params: list = None) -> Tuple[dict, dict]:
+    """Split params dict into dict with matching params and a dict with default values"""
+
+    if unique_params is None:
```

Comment:
It isn't possible to set the default like this, is it?

```py
 def split_params(params: dict, unique_params: list = ["name", "slug"]) -> Tuple[dict, dict]:
```

Reply:
You could do that, but I don't find setting a mutable object as a default value. And also it makes the signature much longer.

Reply:
Thanks for explaining. I'm good with that.